src/main/resources/mailTemplate/email-template.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Confirmation of Interview Schedule on Linchpino</title>
+</head>
+<body>
```

Comment:
The mail template doesn't need HTML, head and body because it does show in  the browser individually